util/chplenv/third_party_utils.py

```diff
@@ -15,10 +15,11 @@
 #
 @memoize
 def default_uniq_cfg_path():
+    (flag, arch) = chpl_arch.get('target', map_to_compiler=True,
```

Comment:
I would go with `(_, arch)` anywhere the `flag` isn't going to be used. Either that or maybe have chpl_arch return a named tuple so that you could just do `chpl_arch.get(...).arch`